homeworks/maksym.selenkov_MaksymSelenkov/5-rxjs/index.html

```diff
@@ -0,0 +1,142 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport"
+            content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>5-rxjs</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <div class="container" data-container>
+            <header class="header" data-header>
+                <span class="header__logo">Logo</span>
+                <div data-burger>
+                    <div class="header__burger-icon"></div>
+                    <div class="header__burger-icon"></div>
+                    <div class="header__burger-icon"></div>
+                </div>
+                <nav class="header__nav" data-navigation>
+                    <a class="header__link" href="">News</a>
+                    <a class="header__link" href="">Contacts</a>
+                    <a class="header__link" href="">About us</a>
+                </nav>
+                <button type="button" class="header__buy" data-buy>Buy now</button>
+            </header>
+            <main class="main" data-main>
```

Comment:
You have never used the 'data-main' selector. So it's better to make your code cleaner and remove it